core/modules/checkout/store/personal/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Module } from 'vuex'
+import RootState from '@vue-storefront/core/types/RootState'
```

Comment:
It should be maybe more named like “profile” - for b2b shops we might want to store more data related to the entity not the exact person; or “account”